Multisig/UI/Transaction/Execution/DisclosureWithContentCell.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  DisclosureWithContentCell.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 11.01.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class DisclosureWithContentCell: UITableViewCell {
```

Comment:
should it be in UI Library?

Reply:
This seems to be used only here now, later maybe